File: internal/sem/testdata/langmatrix/java/Ledger.java
package fixtures;

public class Ledger {
    private int total;

    public int add(int amount) {
        return total + amount;
    }

    public static int ledgerDouble(int amount) {
        Ledger ledger = new Ledger();
        return ledger.add(amount) * 2;
    }
}

File: internal/sem/testdata/langmatrix/php/Ledger.php
<?php

namespace Fixtures;

class Ledger
{
    private int $total = 0;

    public function add(int $amount): int
    {
        return $this->total + $amount;
    }
}

function ledgerDouble(int $amount): int
{
    $ledger = new Ledger();
    return $ledger->add($amount) * 2;
}
